modules/custom_role_iam/variables.tf

```diff
@@ -27,13 +27,14 @@ variable "title" {
 
 variable "base_roles" {
   type        = list(string)
-  description = "List of base predefined roles to use to compose custom role."
+  description = "List of base predefined roles to use to compose custom role. Either base_roles or permissions must be set with some values"
   default     = []
```

Comment:
Can we add a validation check here and to permissions to confirm if either variable is set?